src/Console/DirectiveCommand.php

```diff
@@ -25,16 +29,151 @@ class DirectiveCommand extends LighthouseGeneratorCommand
      */
     protected $type = 'Directive';
 
+    /**
+     * The imports required by the various interfaces, if any.
+     *
+     * @var Collection
+     */
+    protected $imports;
+
     protected function getNameInput(): string
     {
-        return ucfirst($this->argument('name')).'Directive';
+        return ucfirst(trim($this->argument('name'))).'Directive';
```

Comment:
Did you find that calling `trim()` is really necessary? I think that the underlying command class already takes care of that.

Reply:
You know what, I have no idea how that actually got there :thinking:... No, the trim is not necessary.